.grunt-config/widgets-css.js

```diff
@@ -92,11 +91,75 @@ class WidgetsCss {
 			return this.widgetsScssFilesList;
 		}
 
-		this.widgetsScssFilesList = fs.existsSync( this.sourceScssFolder ) ? fs.readdirSync( this.sourceScssFolder ) : [];
+		this.widgetsScssFilesList = this._getStandAloneWidgetsCssFilesList().concat( this._getModulesWidgetsCssFilesList() );
 
 		return this.widgetsScssFilesList;
 	}
 
+	_getStandAloneWidgetsCssFilesList() {
+		const standAloneWidgetData = [],
+			standAloneSidgetsList = fs.existsSync( this.sourceScssFolder ) ? fs.readdirSync( this.sourceScssFolder ) : [];
+
+		standAloneSidgetsList.forEach( ( filename ) => {
+			const widgetName = filename.replace( '.scss', '' );
+
+			standAloneWidgetData.push( {
+				widgetName: widgetName,
+				defaultFilename: this.cssFilePrefix + filename,
+				rtlFilename: this.cssFilePrefix + filename.replace( '.scss', '-rtl.scss' ),
```

Comment:
You can now remove the `getCssFileNames` function for not being used anywhere.

Reply:
Done